### DISCUSSION TITLE
The power shows with a downside

### DISCUSSION THREAD

**Discussion by pepsonEL** [WillCodeForCats/solaredge-modbus-multi#70] [General]
Hi. I start use your integration with solar panels and counter. All looks good but in HA the power shows with a downside ? Why ?
![minus](https://user-images.githubusercontent.com/71824185/183265687-e78c79d0-cbe0-4e09-92c7-ae9a085f397f.jpg)

**Comment by WillCodeForCats** [comment]
On an E+I meter negative means importing power form grid, positive means exporting excess power to the grid.
